code/modules/ai/ai_behaviors/human_mobs/human_mob.dm

```diff
@@ -381,3 +430,15 @@
 
 /datum/ai_behavior/human/suicidal
 	minimum_health = 0
+
+/// Monkeys are locked to saying primitive lines and don't do any text replacement
+/datum/ai_behavior/human/monkey_business
+
+/datum/ai_behavior/human/monkey_business/self_text_replacement(text)
+	return text
+
+/datum/ai_behavior/human/monkey_business/them_text_replacement(text, mob/living/them)
+	return text
+
+/datum/ai_behavior/human/monkey_business/custom_speak(message, cooldown, unique_cooldown_key, unique_cooldown_time, force)
+	INVOKE_ASYNC(mob_parent, TYPE_PROC_REF(/atom/movable, say), pick(GLOB.ai_monkey_lines))
```

Comment:
monkeys have a language. you should not be shitcoding them into saying ook to each other with all this instead of just actually setting the language for them.